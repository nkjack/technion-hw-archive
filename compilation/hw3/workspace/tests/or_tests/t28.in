int foo(){
return 4;
}
void main(){

if(foo()) printi(3);
while(foo()) print("hi");
}
