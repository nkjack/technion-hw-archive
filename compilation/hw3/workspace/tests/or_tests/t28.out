---end scope---
foo - 0 preconditions
line 6: type mismatch
